dkeyserver/App/main.cpp

```diff
@@ -14,9 +14,12 @@
 #include <netdb.h>
 #include <poll.h>
 #include <stdlib.h>
+#include <getopt.h>
 
 #define ENCLAVE_PATH "libenclave-ehsm-dkeyserver.signed.so"
```

Comment:
update the commit msg as:
"
Add domainkey backup support in dkeyserver

In this implementation, if you consider the domainkey backup
support, it suggest to deploy the service into a dkeyserver K8S cluster.
the node in the cluster will be tagged with 2 types:
    - Root: domainkey only generated in this node, it will provide distribution
                service to the other worker nodes inside the dkeyserver cluster.
    - Worker: retrieve the domainkey from Root, then provide dkey-provisioning
                service together to the dkeycache in KMS cluster.
All the nodes are deployed with the same dkeyserver container
image in dkeyserver k8s cluster. And each node could be down(or crashed)
or reboot, unless there’s one is alive in the cluster.

Of cause, if you don't consider the domainkey backup, you can directly
deploy the dkeyserver in a standalone SGX capable node which can continue
to provide the domainkey distribution service to the dkeycache.

test: passed all the unit test
Signed-off-by: hongjingmao <hongjing.mao@intel.com>
"

Reply:
done.